### ISSUE TITLE
Horizontal scrolling (WCAG 1.4.10a)

### ISSUE THREAD

**Issue by VivienDZB** [slidewiki/slidewiki-platform#1229]
**Requesting:** Page content should wrap with a browser window width of 320 CSS pixels (or with a browser window width of 1280 CSS pixels and 400% zoom magnification) so that all information and functions are available without users having to scroll horizontally.

**Explanation:** Visually impaired users often enlarge page content using the zoom function that is available in common desktop browsers. Responsive design using CSS media queries is intended to support web pages with high zoom by dynamically adjusting the page break. The advantage: users only have to scroll in one direction when reading (in Western languages: vertically). If lines do not wrap when zoomed in, users are forced to scroll back and forth horizontally when reading each line, which greatly affects and slows down the recording of the content.

**Issues:**
1. https://slidewiki.org/ 

- Some parts of the content are not fully accessible with a window width of 320px because they are cut off on the right side. Entries in the form fields "E-Mail" and "Password" are not possible or are not displayed, which makes registration difficult. Frames around content sometimes do not adapt to the format, so that there is overlap with the content.
- The registration button cannot be controlled with the keyboard with a window width of 320 px.

2. https://slidewiki.org/presentation/81127-1/overview-of-accessibility/811271/437649-3/?language=en-/slide-719796-3 

- Images partially overlap the content in this view (e.g. page 10). The same thing happens in normal view.

3. a. https://slidewiki.org/deck/127411/_/slide/836159-2/836159-2:1/edit 
b. https://slidewiki.org/deck/127411/_/slide/836159-2/836159-2:1/markdownEdit 

- See homepage.
- You cannot edit slides in this view. The markdown editor is also inaccessible.

See homepage (1.):
4. https://slidewiki.org/search?keywords=accessibility&sort=score 
5. https://slidewiki.org/user/vivien.zschammer/recommendations 
6. https://slidewiki.org/addDeck 
7. https://slidewiki.org/contactus 
8. https://slidewiki.org/deck/127411/test/deck/127411 
9. https://slidewiki.org/accessibility 
10. a. https://slidewiki.org/user/vivien.zschammer/settings/profile
b. https://slidewiki.org/user/vivien.zschammer/stats 

**Solution:**
The size of layout areas can be adapted to different output formats with the help of media queries and flexbox, so that content in the available space increases, decreases or breaks and reacts to zoom levels. The properties width and max-width for labels and input fields should be set so that they adapt to the output format.

**Comment by aoelen** [comment]
Most of the issues w.r.t. to responsive sizing of elements are fixed in https://github.com/slidewiki/slidewiki-platform/pull/1270

Related to: _You cannot edit slides in this view. The markdown editor is also inaccessible._. This is correct, there are a decision made a while ago, that the slide edit buttons are hidden on small screens (mainly meant for disabling editing on smartphones)

I made a few more changes to address some usability issues (all for screen of 320px wide):

Sign in:
![image](https://user-images.githubusercontent.com/1567948/96997488-26e4a980-1532-11eb-831e-2493b2ca5535.png)

Registration (also fixed the captcha sizing issues on other pages where this is used):
![image](https://user-images.githubusercontent.com/1567948/96997606-5c899280-1532-11eb-99af-870bbfa0d9be.png)

Search:
![image](https://user-images.githubusercontent.com/1567948/96997696-89d64080-1532-11eb-89d4-5cb18069c3a8.png)

Recommended decks:
![image](https://user-images.githubusercontent.com/1567948/96997826-cc981880-1532-11eb-91ca-4ff9e3a81276.png)

**Comment by aoelen** [comment]
Keep open until PR is merged
